app/app.py

```diff
@@ -1,7 +1,12 @@
+import os
+import sys
 from callbacks import app
 from layouts import create_layout
 
 
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
You don't have to add this line. Changing `from .config import ...` to `from config import ...` will just works.